egs/multi_en/s5/local/hub4_data_prep.py

```diff
@@ -0,0 +1,242 @@
+#! /usr/bin/env python
+
+# Copyright 2016    Vimal Manohar
+# Apache 2.0.
+
+"""This script prepares the 1996 English Broadcast News (HUB4) corpus.
+https://catalog.ldc.upenn.edu/LDC97S44
+https://catalog.ldc.upenn.edu/LDC97T22
+"""
+
+from __future__ import print_function
+import argparse
+import glob
+import logging
+import os
+import re
+from bs4 import BeautifulSoup
```

Comment:
There is already hub4_spanish recipe that has the data parsing scripts already. Is it necessary to create another one, especially when it has external dependencies?

Reply:
Thanks Yenda for pointing it out. @vimalmanohar , since you're familiar with hub4, do you have an idea whether we can use hub4_spanish's data_prep scripts for english? Thanks

Reply:
Yes, looking at the spanish script I think it is possible with minor modifications. I'll try to update it.

Reply:
That'll be great. Thanks man!! btw, you may want to refer to egs/multi_en/s5/local/g2p/train_g2p.sh about the convention in the header, when copying & modifying scripts.

Reply:
Great, guys